components/tyk-dashboard/templates/service-dash.yaml

```diff
@@ -19,7 +19,7 @@ spec:
   - port: {{ .Values.global.servicePorts.dashboard }}
     targetPort: {{ .Values.global.servicePorts.dashboard }}
     protocol: TCP
-    name: http
+    name: {{ default "http" .Values.global.servicePortName | quote }}
```

Comment:
why dashboard's port name is located in the global whereas others reside in component level?